### ISSUE TITLE
Latest update Version 1.15.3 creates incorrect namespace

### ISSUE THREAD

**Issue by lexman1958** [jaguadoromero/vscode-php-create-class#23]
Hi,

My VS Code install just updated the extension to version 1.15.3. When i create a class now it generate a messy namespace.  Refer to the code generated below:

```
<?php

declare(strict_types=1);

namespace c:\xampp\htdocs\MyCoursesLG\php\mvcframewks\garyclarke\frmwkpro\framework\src\Controller;

class Test
{

}
```

**Comment by jaguadoromero** [comment]
@lexman1958 I just published an update with a fix (1.15.4). It seems that the problem was in the backslash that Windows uses at path separator.

**Comment by lexman1958** [comment]
@jaguadoromero 

Hi, It works now. Thank you so much.
